getaddressmempool.html

```diff
@@ -36,21 +36,6 @@ <h1>getaddressmempool - Zcash 5.5.0 RPC</h1>
 
 Returns all mempool deltas for an address.
 
-WARNING: getaddressmempool is disabled.
```

Comment:
Why did these disappear?

Reply:
I ran a test using `zcash-cli getaddressmempool` and at least it returns some value. I don't know if that warning was related to some version or if it's something that should be maintained